guides/posts/pratham/create-a-text-sentiment-analysis-app/post.md

```diff
@@ -23,11 +23,11 @@ RapidAPI handles all the API development things very effectively. You don’t ne
 
 Go to [RapidAPI Hub](https://rapidapi.com/hub?utm_source=medium.com%2F%40Rapid_API&utm_medium=DevRel&utm_campaign=DevRel) and create a free account.
 
-![sign up page of RapidAPI](https://raw.githubusercontent.com/PrathamKumar14/DevRel-Stack-Data/dev/guides/posts/pratham/create-a-text-sentiment-analysis-app/RapidAPI-homepage.png)
+![sign up page of RapidAPI](https://raw.githubusercontent.com/PrathamKumar14/DevRel-Stack-Data/dev/guides/posts/pratham/create-a-text-sentiment-analysis-app/RapidAPI-Homepage.jpeg)
```

Comment:
The URL of the images should be from this repository.

Reply:
This issue still needs to be fixed.

Reply:
This will get automatically resolved once you merge. As I mentioned, the image is missing because I renamed it.

As you can see, all images are perfect [here](https://github.com/PrathamKumar14/DevRel-Stack-Data/blob/dev/guides/posts/pratham/create-a-text-sentiment-analysis-app/post.md).

Reply:
The URL is still `https://raw.githubusercontent.com/PrathamKumar14/DevRel-Stack-Data` which is your repository. It needs to be `https://raw.githubusercontent.com/RapidAPI/DevRel-Stack-Data`.